baker/SiteBaker.tsx

```diff
@@ -375,6 +378,46 @@ export class SiteBaker {
         this.progressBar.tick({ name: "✅ baked special pages" })
     }
 
+    private async validateGrapherDodReferences() {
+        if (!this.bakeSteps.has("dods") || !this.bakeSteps.has("charts")) return
+
+        const {
+            content: { details },
+        } = await Gdoc.getGdocFromContentSource(GDOCS_DETAILS_ON_DEMAND_ID)
+
+        if (!details) {
+            this.progressBar.tick({
+                name: "✅ no details exist. skipping grapher dod validation step",
+            })
+            return
+        }
+
+        const charts: { slug: string; subtitle: string; note: string }[] =
+            await db.queryMysql(`
+            SELECT config->>'$.slug' as slug, config->>'$.subtitle' as subtitle, config->>'$.note' as note
+            FROM charts
+            WHERE JSON_EXTRACT(config, "$.isPublished") = true
+            AND JSON_EXTRACT(config, "$.subtitle") LIKE "%#dod:%"
+            OR JSON_EXTRACT(config, "$.note") LIKE "%#dod:%"
```

Comment:
Probably best to wrap these two lines into parens.

Reply:
ah, no, what I actually meant here is to wrap the `JSON_EXTRACT() OR JSON_EXTRACT()` into parens.